shacl12-core/index.html

```diff
@@ -6140,23 +6140,23 @@ <h3>sh:closed, sh:ignoredProperties</h3>
 							If <code>$closed</code> is <code>sh:ByTypes</code>, then <code>P</code> is the set of <a>IRI</a> properties
 							that can be reached from the value node via the following algorithm, plus <code>rdf:type</code>:
 							<div style="margin: 20px">
-							<code>
-function collectProperties(S)<br />
-&nbsp;&nbsp;add all IRI properties that can be reached from S via the SPARQL path<br />
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;sh:property/sh:path<br />
-&nbsp;&nbsp;if S is a SHACL instance of rdfs:Class in the shapes graph {<br />
-&nbsp;&nbsp;&nbsp;&nbsp;for each triple in the shapes graph matching (S rdfs:subClassOf ?o)<br />
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;collectProperties(?o)<br />
-&nbsp;&nbsp;&nbsp;&nbsp;for each triple in the shapes graph matching (?s sh:targetClass S)<br />
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;collectProperties(?s)<br />
-&nbsp;&nbsp;}<br/>
-&nbsp;&nbsp;if S is a SHACL instance of sh:NodeShape in the shapes graph<br />
-&nbsp;&nbsp;&nbsp;&nbsp;for each triple in the shapes graph matching (S sh:node ?o)<br />
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;collectProperties(?o)<br />
-<br />
-for each rdf:type T of the value node in the data graph<br />
-&nbsp;&nbsp;collectProperties(T)<br />
-</code></div>
+							<pre class="text">
+function collectProperties(S)
+	add all IRI properties that can be reached from S via the SPARQL path
+			sh:property/sh:path
+	if S is a SHACL instance of rdfs:Class in the shapes graph {
+		for each triple in the shapes graph matching (S rdfs:subClassOf ?o)
+			collectProperties(?o)
+		for each triple in the shapes graph matching (?s sh:targetClass S)
+			collectProperties(?s)
+	}
+	if S is a SHACL instance of sh:NodeShape in the shapes graph
+		for each triple in the shapes graph matching (S sh:node ?o)
+			collectProperties(?o)
+
+for each rdf:type T of the value node in the data graph
+	collectProperties(T)
+</pre></div>
```

Comment:
This is the incantation I mentioned previously. These changes replace the (new) tab-based indents with space-based indents, matching the counts of `&nbsp;` in the previous text.
```suggestion
							<pre class="text">
function collectProperties(S)
  add all IRI properties that can be reached from S via the SPARQL path
      sh:property/sh:path
  if S is a SHACL instance of rdfs:Class in the shapes graph {
    for each triple in the shapes graph matching (S rdfs:subClassOf ?o)
      collectProperties(?o)
    for each triple in the shapes graph matching (?s sh:targetClass S)
      collectProperties(?s)
  }
  if S is a SHACL instance of sh:NodeShape in the shapes graph
    for each triple in the shapes graph matching (S sh:node ?o)
      collectProperties(?o)

for each rdf:type T of the value node in the data graph
  collectProperties(T)
</code></div>
```

Reply:
Ok I have replaced those tabs with spaces, but I didn't use your suggestion because it introduces an incorrect </code> instead of </pre> at the end.